Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -140,34 +156,28 @@ export RGW_POOL_PREFIX=default
     rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
     ```
 
-6. StorageClass will also be created, verify its creation. `ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
+6.  Verify the creation of the storage class depending on the rbd pools and filesystem provided.
+    `ceph-rbd` and `cephfs` would be the respective names for the RBD and CephFS storage classes.
     ```console
     kubectl -n rook-ceph-external get sc
     ```
 
 7. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
 
-### CephCluster example (management)
-
-The following CephCluster CR represents a cluster that will perform management tasks on the external cluster.
-It will not only act as a consumer but will also allow the deployment of other CRDs such as CephFilesystem or CephObjectStore.
-You would need to inject the admin keyring for that.
-
-The corresponding YAML example:
-
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephCluster
-metadata:
-  name: rook-ceph-external
-  namespace: rook-ceph-external
-spec:
-  external:
-    enable: true
-  dataDirHostPath: /var/lib/rook
-  cephVersion:
-    image: quay.io/ceph/ceph:v17.2.6 # Should match external cluster version
-```
+#### Connect to an External Object Store
+
+Create the object store resources:
+1. [External object store](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml), it instruct Rook to consume external gateways.
+2. [Object store user](https://github.com/rook/rook/blob/master/deploy/examples/object-user.yaml), it will create an independent(single bucket) set of user credentials to access the S3 endpoint.
+3. [Storage class for buckets](https://github.com/rook/rook/blob/master/deploy/examples/storageclass-bucket-delete.yaml), create a bucket where a client can read and write objects.
+4. [Object Bucket Claim](https://github.com/rook/rook/blob/master/deploy/examples/object-bucket-claim-delete.yaml), these are the OBCs claims that will create buckets.
+
+!!! hint
+    For more details see the [Object Store topic](../../Storage-Configuration/Object-Storage-RGW/object-storage.md#connect-to-an-external-object-store)
+
```

Comment:
How about adding an example `kubectl create` command with the resources that were mentioned in this section?

Reply:
yup